### PR TITLE
 react router setup is done

### PR DESCRIPTION
**Pull request by RandInaim**
#34  please check the setup for the react router

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
 import React from 'react'
-
+import Home from './components/Home'
+import Start from './components/Start'
 import './App.css'
+import { Route, BrowserRouter as Router, Switch } from 'react-router-dom'
```
Comment (jema28) [inline client/src/App.js]: This should come before the component imports.

File: client/src/App.js [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
 
-function App() {
-  return <>loading</>
+const App = () => {
+	return (
+		<Router>
+			<Switch>
+				<Route exact path="/" component={Start} />
```
Comment (jema28) [inline client/src/App.js]: Aren't these the same thing?

File: client/src/components/SharedComponents/ItemsList/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const home = () => {
```
Comment (jema28) [inline client/src/components/SharedComponents/ItemsList/index.js]: Components should always be capitalised e.g. Home

File: client/package.json [hunk 4]
```diff
@@ -1,46 +1,38 @@
...
-    "eslint-config-wesbos": "0.0.19",
-    "eslint-plugin-html": "^5.0.5",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "^7.14.3",
-    "eslint-plugin-react-hooks": "^1.7.0",
-    "prettier": "^1.18.2"
-  }
+	"name": "hebron",
```
Comment (jema28) [inline client/package.json]: Can you make sure your eslint/prettier settings are the same as Saja's.

File: client/src/components/Start/index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import { Link, BrowserRouter as Router } from 'react-router-dom'
```
Comment (FarahZaqout) [inline client/src/components/Start/index.js]: This import is not used. It should be removed.

**Pull request merged**